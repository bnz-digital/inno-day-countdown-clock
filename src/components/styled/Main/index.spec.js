import React from 'react'
import { shallow } from 'enzyme'

import StyledMain from '.'

describe('components:styled:Main', () => {
  it('renders <StyledMain/> correctly', () => {
    expect(toJson(shallow(<StyledMain />))).toMatchSnapshot()
  })
})
