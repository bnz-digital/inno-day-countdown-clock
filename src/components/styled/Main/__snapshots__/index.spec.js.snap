// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:styled:Main renders <StyledMain/> correctly 1`] = `
.c0 {
  height: 100%;
  width: 100%;
  display: grid;
  grid-template-areas: "message";
  grid-template-rows: 1fr;
}

<main
  className="c0"
/>
`;
